modules/claims_api/spec/lib/claims_api/v2/error/lighthouse_error_handler_spec.rb

```diff
@@ -74,6 +81,24 @@ def raise_expired_token_signature
     end
   end
 
+  it 'returns a 400, Bad Request, in line with LH standards for an array of errors' do
+    mock_ccg(scopes) do |auth_header|
+      request.headers.merge!(auth_header)
+
+      get :raise_bad_request
+
+      expect(response).to have_http_status(:bad_request)
+
+      parsed_body = JSON.parse(response.body)
+      expect(parsed_body['errors'].size).to eq(1)
+      expect(parsed_body['errors'][0]['title']).to eq('Bad Request')
+      expect(parsed_body['errors'][0]['detail']).to eq('Test 400')
+      expect(parsed_body['errors'][0]['status']).to eq('400')
+      expect(parsed_body['errors'][0]['status']).to be_a(String)
+      expect(parsed_body['errors'][0]['source'].to_s).to include('{"pointer"=>')
```

Comment:
- More drumming up thought - stimulated from this spec, not a needed change in this PR

I'd like to take the stack out for these. be a simple lift here but it would update a lot of documentation (all of these particular 422's and now the 400's from 526 on) so probably best in another ticket.

`render_error` for 422 and 400 both provide this but they're just a stacktrace which is not useful nor needed. I believe the solution is to remove [this line](https://github.com/department-of-veterans-affairs/vets-api/blob/547843188ee22cce11701369e8e0e7ee43c30ca2/modules/claims_api/lib/claims_api/v2/error/lighthouse_error_handler.rb#L50) from the `render_error` method... that could impact [the above](https://github.com/department-of-veterans-affairs/vets-api/blob/master/modules/claims_api/lib/claims_api/v2/error/lighthouse_error_handler.rb#L15-L20) as well, but I don't think those would be useful nor beneficial either. I think the only portion we have that is useful comes from the [JsonApiMissingAttribute](https://github.com/department-of-veterans-affairs/vets-api/blob/547843188ee22cce11701369e8e0e7ee43c30ca2/modules/claims_api/lib/claims_api/common/exceptions/lighthouse/json_disability_compensation_validation_error.rb#L24) and/or JSON Schemer raises them immediately? @rockwellwindsor-va - you were deeply nested in this build so just curious your take. These 400's and 422's on the portal that said "no participant ID" with a stack aren't pretty nor useful - thinking we can nip this with a quick cleanup ticket? Thoughts? Do you see changes _outside of the above_ that may cause significantly more workload to do this? If not I think it's cleanup and generate docs, right?

Reply:
New ticket: https://jira.devops.va.gov/browse/API-36965

Reply:
@FonzMP  I think the main reason for that addition had to do with the lighthouse standards.  For a 400 that might not be applicable, which means we could give it its own treatment, I think for 422's though it is?  The standards say 
```
If an error is in the request, a source field should point to it.
```